packages/scaffold-config/src/frameworks.ts

```diff
@@ -38,7 +41,6 @@ export async function isDependencyInstalled (dependency: Cypress.CypressComponen
     return {
       dependency,
       detectedVersion: pkg.version,
-      loc,
       satisfied,
     }
   } catch (e) {
```

Comment:
Total nitpick, but for this debug logging is a failure to lookup a dependency really an "error"? We have the ability to differentiate an error (filesystem permissions, etc) from a "dependency is missing" condition. Could report each differently by doing an early return above rather than throwing an Error to be caught here

Reply:
Good point, I updated this so that it just logs to debug that we couldn't resolve the dependency - no need to throw an error